README.md

```diff
@@ -89,30 +89,28 @@ consumer = KafkaConsumer(servers='localhost:9092',
 consumer.thread()
 ```
 
-### Usage with SQS:
+### Usage for SQS:
 
-> Installing this library will also install `boto3`
+To use the eventcore.sqs feature, please run `pip install eventcore[kafka]`.
```

Comment:
evencore[sqs]

Reply:
Fixed in a06cc2d30d133fae11c71a19d82137152e1d2972